README.md

```diff
@@ -57,9 +57,9 @@ Given those basics, the OS and backend specific instructions are below.
 #### Generic
 
 1. Install backend:
-    - If you want to use NVidia graphics cards Install [CUDA](https://developer.nvidia.com/cuda-zone) and [cuDNN](https://developer.nvidia.com/cudnn).
+    - If you want to use NVIDIA graphics cards install [CUDA](https://developer.nvidia.com/cuda-zone) and [cuDNN](https://developer.nvidia.com/cudnn).
     - If you want to use AMD graphics cards install OpenCL.
```

Comment:
This is probably the right place to mention the `dx12` backend for Win10 10.0.18363 and new enough drivers.